MultiSigWalletOverride/migrations/2_deploy_contracts.js

```diff
@@ -4,25 +4,25 @@ const MultisigWalletFactory = artifacts.require('MultiSigWalletWithDailyLimitFac
 const fs = require('fs');
 const path = require('path');
 
-module.exports = deployer => {
+module.exports = async (deployer, [_deployerAddress]) => {
   const args = process.argv.slice();
-  if (process.env.DEPLOY_FACTORY){
-    deployer.deploy(MultisigWalletFactory);
-    console.log("Factory with Daily Limit deployed");
-  } else if (args.length < 5) {
-    console.error("Multisig with daily limit requires to pass owner " +
-      "list, required confirmations and daily limit");
-  } else if (args.length < 6) {
-    deployer.deploy(MultisigWalletWithoutDailyLimit, args[3].split(","), args[4]).then(function() {
+  const accountsIndex = args.indexOf('--accounts');
+  const confirmationsIndex = args.indexOf('--confirmations');
+  const numberOfConfirmations = args[confirmationsIndex + 1];
+  const accounts = args[accountsIndex + 1].split(",");
+  if (accountsIndex === -1 || confirmationsIndex === -1) {
+    console.error('ABORTED. Use: --accounts 0xasdf,0xfdsa --confirmations 2');
+    process.exit(1); 
+  } else {
+    console.log(`Accounts: ${accounts}`);
+    console.log(`Confirmations: ${numberOfConfirmations}`);
+    deployer.deploy(MultisigWalletWithoutDailyLimit, accounts, numberOfConfirmations).then(function() {
```

Comment:
Minor: Using `await` here might be better / easier to read.
As it is, if the deploy fails you'll still get a `Wallet deployed` log, which could be confusing.

Either use `await` or move the `console.log("Wallet deployed");` into the `then` block